x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_tables/shared_columns_tests.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  mockHttpValues,
+  mockKibanaValues,
+  mockFlashMessageHelpers,
+} from '../../../../../__mocks__';
+import '../../../../__mocks__/engine_logic.mock';
+
+import { ReactWrapper } from 'enzyme';
+
+import { nextTick } from '@kbn/test/jest';
+
+export const runActionColumnTests = (wrapper: ReactWrapper) => {
```

Comment:
I wasn't sure if I could export a reusable function that ran a set of tests and was super excited to find out that I could :) this DRY's out a huge amount of copied and pasted tests between the RecentQueriesTable and AnalyticsTable components.